controller.js

```diff
@@ -1,15 +1,21 @@
-
 const endpointsJson = require("./endpoints.json");
-const topicsJson = require("./db/data/development-data/topics");
-const {getAllTopicsData} = require("./models")
+const {getAllTopicsData, getArticleDataById} = require("./models")
 
 function getEndpoints(req, res, next){
     res.status(200).send({endpoints: endpointsJson});
 }
 
 function getAllTopics(req, res, next){
-    getAllTopicsData().then((rows) =>{
-        res.status(200).send({topics:rows})
+    getAllTopicsData().then((topicsData) =>{
+        res.status(200).send({topics:topicsData})
```

Comment:
considering you receive it and send it back one line later, could you refactor it so that the keyname is the same as the value?